modules/ray_tracing/doc/content/modules/ray_tracing/examples/line_integrals.md

```diff
@@ -14,17 +14,17 @@ We begin with the standard "simple diffusion" problem:
 For this problem, we seek the value of the integral
 
 !equation
-\int_L u_h~dr, \quad L = \{\vec{r}_1 + t\vec{r}_2 \mid t \in [0, 1]\}
+\int_L u_h(\vec{r})~dr, \quad L = \{\vec{r}_1 + t\vec{r}_2 \mid t \in [0, 1]\}
```

Comment:
I understand now what you mean by u_h.  I think you should either:

- state "u_h is the finite-element solution for u"
- or simply have "u".  I hope that MOOSE users will understand that MOOSE results are subject to finite-element discretisation approximations, so won't expect their line integrals to always agree with the analytic solution.

Reply:
👍🏼